src/esp/assets/ResourceManager.cpp

```diff
@@ -89,7 +91,9 @@ ResourceManager::ResourceManager(
       importerManager_("nonexistent")
 #endif
 {
-
+#ifdef ESP_BUILD_WITH_VHACD
+  interfaceVHACD = VHACD::CreateVHACD();
```

Comment:
Doesn't this need to be freed in some way? I don't see any code to do so in the destructor.

Reply:
Hmm I'm not quite sure how to do this, or if it's needed. I tried "delete interfaceVHACD" in the destructor but it looks like the destructor for VHACD interface is protected. Would changing interfaceVHACD to be a ptr, and then just setting it to be a nullptr work?

Reply:
> Would changing interfaceVHACD to be a ptr, and then just setting it to be a nullptr work?

Setting a raw pointer to null doesn't free any memory, no. Using a smart pointer for this won't work either because the destructor is protected.

I did some digging and it seems you have to call `interfaceVHACD->Clean()` and interfaceVHACD->Release()` in order to clean up. In my opinion that's a *really bad* interface (did the author of that thing ever hear about destructors?!), but what can we do, sigh.

Reply:
Haha, yeah, Scott and I chatted about this a couple hours ago. We arrived at the same conclusion. 🤷